l10n/template.ftl

```diff
@@ -0,0 +1,288 @@
+obs-assessment = Developed by Mozilla, the { -brand-name-obs } performs an in-depth assessment of a site’s HTTP headers and other key security configurations.
+obs-scanning = Its automated scanning process provides developers and website administrators with detailed, actionable feedback, focusing on identifying and addressing potential security vulnerabilities.
+obs-security = The tool is instrumental in helping developers and website administrators strengthen their sites against common security threats in a constantly advancing digital environment.
+obs-mdn = The { -brand-name-obs } provides effective security insights, guided by Mozilla's expertise and commitment to a safer and more secure internet and based on well-established trends and guidelines.
+compat-loading = Loading…
+compat-browser-version-date = { $browser } { $version } – Released { $date }
+compat-browser-version-released = Released { $date }
+compat-link-report-issue = Report problems with this compatibility data
+compat-link-report-issue-title = Report an issue with this compatibility data
+compat-link-report-missing-title = Report missing compatibility data
+compat-link-report-missing = Report this issue
+compat-link-source = View data on GitHub
+compat-link-source-title = File: { $filename }
+compat-deprecated = Deprecated
+compat-experimental = Experimental
+compat-nonstandard = Non-standard
+compat-no = No
+compat-support-full = Full support
+compat-support-partial = Partial support
+compat-support-no = No support
+compat-support-unknown = Support unknown
+compat-support-preview = Preview browser support
+compat-support-prefix = Implemented with the vendor prefix: { $prefix }
+compat-support-altname = Alternate name: { $altname }
+compat-support-removed = Removed in { $version } and later
+compat-support-see-impl-url = See <a data-l10n-name="impl_url">{ $label }</a>
+compat-support-flags =
+    { NUMBER($has_added) ->
+        [one] From version { $version_added }
+       *[other] { "" }
+    }{ $has_last ->
+        [one]
+            { NUMBER($has_added) ->
+               *[zero] Until { $versionLast } users
+                [one] { " " }until { $versionLast } users
+            }
+       *[zero]
+            { NUMBER($has_added) ->
+               *[zero] Users
+                [one] { " " }users
+            }
+    }
+    { " " }must explicitly set the <code data-l10n-name="name">{ $flag_name }</code>{ " " }
+    { $flag_type ->
+       *[preference] preference
+        [runtime_flag] runtime flag
+    }{ NUMBER($has_value) ->
+        [one] { " " }to <code data-l10n-name="value">{ $flag_value }</code>
+       *[other] { "" }
+    }{ "." }
+    { $flag_type ->
+        [preference] To change preferences in { $browser_name }, visit { $browser_pref_url }.
+       *[other] { "" }
+    }
```

Comment:
This is untranslatable. Even in English I barely understand what a final string would look like, so compounding translations with different word order, you'll have a very difficult time to get something comprehensible out of this.

This should be split into multiple strings. E.g.
`string-a = From version { version }`
`string-b = From version { version } until {version} users`
ETc.